examples/outcomes/1/outcomes5.json

```diff
@@ -0,0 +1 @@
+{"timestamp":"2023-03-28T22:51:00.000000Z","project_id":5466470,"outcome":3,"reason":"project_id","source":"pop-us","category":1,"quantity":1}
```

Comment:
Bit of a nitpick but would you mind giving these files more useful names? e.g. This one can be `outcomes_pops.json` or whatever. So people reading these files have a better idea of what they are.